api/v2.0/swagger.yaml

```diff
@@ -1755,6 +1755,59 @@ paths:
           $ref: '#/responses/401'
         '500':
           $ref: '#/responses/500'
+  /auditlog-exts:
```

Comment:
nit:  IMO it should be `auditlogs-ext`, if you agree, please also update the code and other resources.